developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,107 @@
 title: "Accounts"
 id: "basics-accounts"
 ---
-An account represents a resource on the Aptos Blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources.
 
-# Introduction
+# Accounts
 
-The state of each account comprises both code and data:
+An account on the Aptos Blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes on access control and scarcity in its representation of an asset.  
 
-- **Code**: Move modules contain code (type and procedure declarations), but they do not contain data. The module's procedures encode the rules for updating the blockchain's global state.
-- **Data**: Move resources contain data but no code. Every resource value has a type that is declared in a module published in the blockchain's distributed database.
+Each account on the Aptos Blockchain is identified by a 32-byte account address. An account can store data and it stores this data in resources. The initial resource is the account data itself (authentication key and sequence number). Additional resources like the currency or NFTs are added after creating the account. 
```

Comment:
the currency -> currency

It might be good to include a code block below showing what an account address looks like.